anger.js

```diff
@@ -103,7 +123,16 @@ function anger (opts) {
   function triggerSender (sender) {
     const uid = opts.trigger(sender)
     if (++totalRequests === opts.requests && tail) setTimeout(complete, tail)
-    map.set(uid, { start: process.hrtime(), sender: sender, expectedResponses: clients.length, id: uid })
+    let replyTimeout
+    if (timeout) replyTimeout = setTimeout(handleTimeout(uid), timeout)
```

Comment:
I don't think standard style allows for no braces around if, 

do we not have linting set up for this project?